### PR TITLE
Adds rubber ducky gun

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Is it a rubber ducky or a gun? find out since its available as maints loot
## Why It's Good For The Game

FUNNI
## Images of changes

Video is outdated, chance for shot is now 2%.

https://user-images.githubusercontent.com/77684085/229166636-8467e72b-cb63-432c-b4a0-bc0eb421cd24.mp4

## Testing

video above
## Changelog
:cl:
add: Added a new gun to maints loot
/:cl:

### CODE REVIEW

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -58,6 +58,7 @@
 	//port:					5  items,  5 spots 0 extra (08/08/2014)
 	loot = list(
 				/obj/item/bodybag = 10,
+				/obj/item/gun/energy/disabler/ducky = 10,
```
Comment (hal9000PR) [inline code/game/objects/effects/spawners/lootdrop.dm:61]: increase this

Comment (Henri215) [reply code/game/objects/effects/spawners/lootdrop.dm:61]: Done Mr Paradise

**Comment by Gatchapod** [comment]
Can't wait to KoS someone for holding a rubber ducky because armed and dangerous clause lets me to.

**Review by farie82 — APPROVED** [review]

File: code/modules/projectiles/guns/energy/stun.dm [hunk 2]
```diff
@@ -80,3 +80,23 @@
...
+	desc = "A plastic yellow duck with a trigger below the base. 'DO NOT SQUEEZE' is inscribed on it's back."
+	icon = 'icons/obj/watercloset.dmi'
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+	icon_state = "rubberducky"
+	item_state = "rubberducky"
+
+/obj/item/gun/energy/disabler/ducky/newshot()
+	if(prob(98))
+		atom_say(pick("Stop touching me!", "I am a duck not a gun!", "Stop squeaking me, its useless!", "You still think i can shoot something?", "I hate you.", "I miss my rubber ducky family.", "I am not shooting anything!", "Still thinking i am a gun?!", "This is not a trigger, its my leg!"))
```
Comment (AffectedArc07) [inline code/modules/projectiles/guns/energy/stun.dm:95]:
```suggestion
		atom_say(pick("Stop touching me!", "I am a duck not a gun!", "Stop squeaking me, its useless!", "You still think I can shoot something?", "I hate you.", "I miss my rubber ducky family.", "I am not shooting anything!", "Still thinking I am a gun?!", "This is not a trigger, its my leg!"))
```

**Pull request merged**